apps/aesophia/src/aeso_ast_infer_types.erl

```diff
@@ -1525,7 +1525,8 @@ pp({qid, _, Name}) ->
 pp({con, _, Name}) ->
     Name;
 pp({uvar, _, Ref}) ->
-    ["?" | lists:sublist(base58:binary_to_base58(term_to_binary(Ref)), 43, 3)];
+    %% Show some unique representation
+    ["?u" | integer_to_list(erlang:phash2(Ref, 16384)) ];
```

Comment:
I expect this changes slightly the error reason in response body of `/debug/contracts/code/compile`. Though this is backward compatible - ok for 1.2.0 cc @noandrea .
May be worth a line in release notes.